app/controllers/maintenance_tasks/application_controller.rb

```diff
@@ -8,10 +8,14 @@ class ApplicationController < MaintenanceTasks.parent_controller.constantize
     BULMA_CDN = "https://cdn.jsdelivr.net"
 
     content_security_policy do |policy|
+      policy.style_src(
```

Comment:
Is this change necessary? We changed from `style_src` -> `style_src_elem` in https://github.com/Shopify/maintenance_tasks/pull/1195 to prevent CSP conflicts when the parent application uses `style_src_elem` directives, since we fall back to `style_src`.